cpp_macros/src/lib.rs

```diff
@@ -176,7 +205,8 @@ pub fn expand_internal(input: TokenStream) -> TokenStream {
     let source = input.to_string();
```

Comment:
One of the awesome things about the new `proc_macro2` is that we don't have to do this to-string dance anymore :-) - Just call `syn::parse<cpp_common::Closure>(input)`. Syn even provides an awesome `parse_macro_input` macro which even does error reporting for us! https://docs.rs/syn/0.15.3/syn/macro.parse_macro_input.html